src/components/CreateAddress/AddressGenerator.jsx

```diff
@@ -191,8 +253,7 @@ function mapStateToProps(state) {
 }
 
 const mapDispatchToProps = {
-  sortPublicKeyImporters,
-  setMultisigAddress,
+  ...publicKeyImporterActions,
```

Comment:
I can see that point and I'm happy to change it to the pattern followed in the other imports (i.e. `import action as actionImport from 'action';`) in order to avoid the `no-shadow` linting rule.

I'm not sure of best practices but I think it makes sense to limit what you import regardless.

Reply:
I agree with being more explicit about what we're importing. I've also seen the `import action as actionImport from 'action';` used so we at least wouldn't be going against any existing best practices in using that pattern